model/dao/TaskDAO/PostgreSQLTaskDAO.php

```diff
@@ -568,6 +568,51 @@ public function getGlobalTaskReport(DateTime $initDate = NULL, DateTime $endDate
 
     }
 
+    /** Weekly hours worked on project by users for PostgreSQL
+     *
+     * @param ProjectVO $projectVO
+     * @param DateTime|null $initDate
+     * @param DateTime|null $endDate
+     * @param null $groupField2
+     * @param null $groupField3
```

Comment:
If you aren't using the parameters `groupField2` and 3 at all, please remove them from the method.